<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Qt5GUI01Class</class>
 <widget class="QWidget" name="Qt5GUI01Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>803</width>
    <height>521</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Qt5GUI01</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" alignment="Qt::AlignBottom">
    <widget class="QGroupBox" name="groupBox">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item alignment="Qt::AlignBottom">
       <widget class="QRadioButton" name="radioButton">
        <property name="text">
         <string>선</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignBottom">
       <widget class="QRadioButton" name="radioButton_2">
        <property name="text">
         <string>사각형</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignBottom">
       <widget class="QRadioButton" name="radioButton_3">
        <property name="text">
         <string>타원</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1" alignment="Qt::AlignBottom">
    <widget class="QGroupBox" name="groupBox">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item alignment="Qt::AlignBottom">
       <widget class="QRadioButton" name="radioButton_7">
        <property name="text">
         <string>검정</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignBottom">
       <widget class="QRadioButton" name="radioButton_6">
        <property name="text">
         <string>하양</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignBottom">
       <widget class="QRadioButton" name="radioButton_8">
        <property name="text">
         <string>빨강</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignBottom">
       <widget class="QRadioButton" name="radioButton_5">
        <property name="text">
         <string>초록</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignBottom">
       <widget class="QRadioButton" name="radioButton_4">
        <property name="text">
         <string>파랑</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="qt5gui01.qrc"/>
 </resources>
 <connections/>
</ui>
